components/tensorflow-notebook-image/Dockerfile

```diff
@@ -185,7 +185,7 @@ RUN jupyter nbextension enable --py widgetsnbextension --sys-prefix
 RUN curl -L -o bazel.sh https://github.com/bazelbuild/bazel/releases/download/0.11.1/bazel-0.11.1-installer-linux-x86_64.sh && chmod a+x ./bazel.sh && ./bazel.sh && rm ./bazel.sh
 SHELL ["/bin/bash", "-c"]
 
-RUN git clone https://github.com/tensorflow/models.git /home/$NB_USER/tensorflow-models && git clone https://github.com/tensorflow/benchmarks.git /home/$NB_USER/tensorflow-benchmarks
+RUN git clone https://github.com/tensorflow/models.git /home/$NB_USER/tensorflow-models && git clone https://github.com/tensorflow/benchmarks.git /home/$NB_USER/tensorflow-benchmarks && chown -R $NB_USER /home/$NB_USER/
```

Comment:
```
# du -sh models/
934M	models/
```
Almost a Gb right there. I don't think we want to clone these do we but rather just grab the [tarballs](https://github.com/tensorflow/models/releases)?

```
# ll -h v1.8.0.tar.gz
-rw-r--r--. 1 root root 551K Apr  3 18:01 v1.8.0.tar.gz
```